### PR TITLE
Lightning Support in CLI

### PR DESCRIPTION
**Pull request by martindale**
Adding Lightning support to the CLI.

**Notable Changes**
- [ ] `@fabric/core` now has a TypeScript declaration

### CODE REVIEW

**Comment by martindale** [comment]
Next streak of updates landing here

**Review by casey-bowman — COMMENTED** [review]
Should there be a file at contract/mount.js?

**Comment by casey-bowman** [comment]
It appears that the [RTL c-lightning-REST API](https://github.com/Ride-The-Lightning/c-lightning-REST/blob/master/docs/invoice.md) does not include a parameter for the payment hash when creating an invoice via invoice/genInvoice. We'll need that _if_ we use this for the swap.

**Comment by martindale** [comment]
> Should there be a file at contract/mount.js?

Yep!  Just added in the latest commit.

**Comment by martindale** [comment]
> It appears that the [RTL c-lightning-REST API](https://github.com/Ride-The-Lightning/c-lightning-REST/blob/master/docs/invoice.md) does not include a parameter for the payment hash when creating an invoice via invoice/genInvoice. We'll need that _if_ we use this for the swap.
> 
> Noticed the use of this API in services/lightning.js

Moved from CREST to direct UNIX sockets, specify `mode: 'socket'` and `path:` to the appropriate UNIX socket file.

File: IDENTITY.md [hunk 1]
```diff
@@ -23,6 +23,21 @@ Send authenticated HTTP requests with the following headers:
...
+
+## Bridging
+Identities from external networks can be "attested" to within Fabric.
+
+### Labels
+For example, Discord users may run the `!fabric assert <pubkey>` command in any room [an attestation bot][attestation-bot] is present:
+
+```
+martindale: !fabric assert 
+ExampleBot: Please run `fabric prove feebadee` and provide the output signature to the command: `!fabric prove <signature>`
```
Comment (martindale) [inline IDENTITY.md:38]: TODO: add `fabric prove` to CLI

**Pull request merged**